### PR TITLE
feat(breaking): support Unichain Mainnet on v2 and v3

### PR DESCRIPTION
**Pull request by xrsv**
## PR Scope

Adds support for Unichain Mainnet enums/addresses on sdk-core
Introduces a breaking change for Unichain Sepolia enum name

## How Has This Been Tested?

Will test in routing/SOR

## Are there any breaking changes?

Yes - Introduces a breaking change for Unichain Sepolia enum name from `ASTROCHAIN_SEPOLIA` to `UNICHAIN_SEPOLIA`

### CODE REVIEW

File: sdks/sdk-core/src/addresses.ts [hunk 1]
```diff
@@ -292,6 +294,16 @@ const ASTROCHAIN_SEPOLIA_ADDRESSES: ChainAddresses = {
   v4QuoterAddress: '0x2cfa87651D3AB05Bc59E325E5eaC8495CF34cE55',
 }
 
+const UNICHAIN_ADDRESSES: ChainAddresses = {
+  v3CoreFactoryAddress: '0x1f98400000000000000000000000000000000003',
+  multicallAddress: '0xb7610f9b733e7d45184be3a1bc966960ccc54f0b',
```
Comment (jsy1218) [inline sdks/sdk-core/src/addresses.ts:299]: multical address is unichain sepolia address. looks like we haven't deployed [deploy-v3](https://github.com/Uniswap/deploy-v3)

File: sdks/sdk-core/src/chains.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ export enum ChainId {
   ZKSYNC = 324,
   WORLDCHAIN = 480,
   ASTROCHAIN_SEPOLIA = 1301,
```
Comment (jsy1218) [inline sdks/sdk-core/src/chains.ts:27]: i've been thinking whether we should rename ASTROCHAIN_SEPOLIA to UNICHAIN_SEPOLIA, now we are here. Then make it a breaking change, by titling PR breaking(sdk-core): ...

File: sdks/sdk-core/src/addresses.ts (hunk 1 above)
Comment (jsy1218) [reply sdks/sdk-core/src/addresses.ts:299]: im wrong, they are from mainnet

File: sdks/sdk-core/src/chains.ts (hunk 2 above)
Comment (dannydaniil) [reply sdks/sdk-core/src/chains.ts:27]: agreed!

**Review by dannydaniil — APPROVED** [review]
lgtm, assuming the addresses have been checked

File: sdks/sdk-core/src/chains.ts (hunk 2 above)
Comment (xrsv) [reply sdks/sdk-core/src/chains.ts]: updated

**Review by cgkol — APPROVED** [review]

**Pull request merged**